app/parse.py

```diff
@@ -15,9 +26,166 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+_driver: WebDriver | None = None
```

Comment:
Using a global variable for the WebDriver instance (`_driver`) can lead to potential issues, especially in larger applications or when running multiple tests concurrently. Consider using dependency injection or passing the driver explicitly to functions that need it.